test/any/client/api_matrix-test.es6.js

```diff
@@ -4,81 +4,67 @@
 'use strict';
 
 describe('ApiMatrix', function() {
+  let Release;
+  let WebInterface;
+  let Junction;
+  let ApiMatrix;
   let apiMatrix;
+
   let release = function(browserName, browserVersion,
     osName, osVersion) {
-      return org.chromium.apis.web.Release.create({
+      return Release.create({
         browserName,
         browserVersion,
         osName,
         osVersion,
       });
     };
   let webInterface = function(interfaceName, apiName) {
-    return org.chromium.apis.web.WebInterface.create({
-      interfaceName,
-      apiName,
-    });
+    return WebInterface.create({interfaceName, apiName});
   };
   let releaseAPI = function(browserName, browserVersion,
-    osName, osVersion, interfaceName, apiName) {
-      return org.chromium.apis.web.ReleaseWebInterfaceJunction.create({
-        sourceId: `${browserName}_${browserVersion}_${osName}_${osVersion}`,
-        targetId: `${interfaceName}#${apiName}`,
-      });
-    };
-  beforeEach(function() {
-    let releaseDAO = foam.dao.EasyDAO.create({
-      name: 'releaseDAO',
-      of: org.chromium.apis.web.Release,
-      daoType: 'MDAO',
-    });
-    let interfaceDAO = foam.dao.EasyDAO.create({
-      name: 'interfaceDAO',
-      of: org.chromium.apis.web.WebInterface,
-      daoType: 'MDAO',
+      osName, osVersion, interfaceName, apiName) {
+    return Junction.create({
+      sourceId: `${browserName}_${browserVersion}_${osName}_${osVersion}`,
+      targetId: `${interfaceName}#${apiName}`,
     });
-    let releaseApiDAO = foam.dao.EasyDAO.create({
-      name: 'releaseApiDao',
-      of: org.chromium.apis.web.ReleaseWebInterfaceJunction,
-      daoType: 'MDAO',
-    });
-    releaseApiDAO.put(releaseAPI('Chrome', '55', 'Windows', '10',
-      'Array', 'find'));
-    releaseApiDAO.put(releaseAPI('Chrome', '55', 'Windows', '10',
-      'Audio', 'stop'));
-    releaseApiDAO.put(releaseAPI('Edge', '14', 'Windows', '10',
-      'Array', 'find'));
-    releaseApiDAO.put(releaseAPI('Edge', '14', 'Windows', '10',
-      'Audio', 'play'));
-    releaseApiDAO.put(releaseAPI('Safari', '10', 'OSX', '601',
-      'ApplePay', 'about'));
-    releaseApiDAO.put(releaseAPI('Safari', '10', 'OSX', '601',
-      'Audio', 'play'));
-    releaseApiDAO.put(releaseAPI('Safari', '10', 'OSX', '601',
-      'Audio', 'stop'));
-    releaseApiDAO.put(releaseAPI('Safari', '10', 'OSX', '601',
-      'Array', 'find'));
+  };
+
+  beforeEach(function() {
+    Release = foam.lookup('org.chromium.apis.web.Release');
+    WebInterface = foam.lookup('org.chromium.apis.web.WebInterface');
+    Junction = foam.lookup('org.chromium.apis.web.ReleaseWebInterfaceJunction');
+    ApiMatrix = foam.lookup('org.chromium.apis.web.ApiMatrix');
+
+    let container = global.getReleaseApiContainer();
```

Comment:
Better name might be createReleaseApiContainer, because it creates a new one every time.